blockchain/storage/leveldb/leveldb.go

```diff
@@ -6,6 +6,12 @@ import (
 	"github.com/0xPolygon/polygon-edge/blockchain/storage"
 	"github.com/hashicorp/go-hclog"
 	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/opt"
+)
+
+const (
+	DefaultCache   = int(256)
+	DefaultHandles = int(256)
```

Comment:
Casts are not necessary. Also does it make sense to leave just a single constant (e.g. `DefaultCache`)? Why do we need two of them with the same values?
```suggestion
	DefaultCache   = 256
	DefaultHandles = 256
```

Reply:
Both values are used for options, and they determine the way how lbd files are created. That's why I left two constants. It's the same value because that's the default value on go-ethereum. Probably this can be a part of some configuration in the future.